### PR TITLE
fix for Atom v1.13.0

### PR DESCRIPTION
**Pull request by SATOFish**
This is fix old-syntax in index.less. 　It corresponds to Atom v1.13.0.

### CODE REVIEW

**Comment by craigmdennis** [comment]
This is still an issue...

**Comment by FNGR2911** [comment]
Yeah, one minute before cloning I saw your PR too. Works great. Thank you!

**Comment by phil-r** [comment]
Fix for #76

File: index.less [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import 'syntax-variables';
 
-atom-text-editor, :host {
+atom-text-editor, atom-text-editor {
```
Comment (dspdesign) [inline index.less]: Why not just `atom-text-editor {`?

Comment (SATOFish) [inline index.less]: I overlooked that. Thank you 💕

**Comment by tavorag** [comment]
Please update the pull request with the changes suggested

**Comment by tavorag** [comment]
Merge plz!

**Comment by kevinsawicki** [comment]
Thanks, published in 0.19.0

**Comment by tavorag** [comment]
Thanks to you!

**Pull request merged**